src/io/flutter/logging/FlutterLogView.java

```diff
@@ -45,20 +72,26 @@
     public SimpleTextAttributes style(@Nullable FlutterLogEntry entry, int attributes) {
       if (showColors && entry != null) {
         final FlutterLog.Level level = FlutterLog.Level.forValue(entry.getLevel());
-        // TODO(quangson91): use color from FlutterLogColorPage
         if (level != null) {
-          switch (level) {
-            case FINER:
-              return SimpleTextAttributes.GRAY_ATTRIBUTES;
-            case WARNING:
-              return ORANGE_ATTRIBUTES;
-            case SEVERE:
-              return SimpleTextAttributes.ERROR_ATTRIBUTES;
-          }
+          return getTextAttributesByLogLevel(level);
         }
       }
       return SimpleTextAttributes.REGULAR_ATTRIBUTES;
     }
+
+    @NotNull
+    private SimpleTextAttributes getTextAttributesByLogLevel(@NotNull Level level) {
+      try {
+        final TextAttributesKey key = LOG_LEVEL_TEXT_ATTRIBUTES_KEY_MAP.get(level);
+        final Color color = globalEditorColorsScheme.getAttributes(key).getForegroundColor();
+        return new SimpleTextAttributes(SimpleTextAttributes.STYLE_PLAIN, color);
+      }
+      catch (Exception e) {
+        // Should never go here.
+        e.printStackTrace();
```

Comment:
Let's add a logger and log this instead of printing a stack trace.

Something like:

    private static final Logger LOG = Logger.getInstance(FlutterLogView.class);

and then:

    LOG.error(e);

?

Reply:
I updated.
Thanks!